test/verify/check-system-services

```diff
@@ -1128,6 +1128,12 @@ Where=/fail
         b.click(".pf-c-toolbar__content > .pf-c-toolbar__item > button:contains('Clear all filters')")
         b.wait_js_cond('window.location.hash == "#/?type=service&owner=system"')
 
+        # Test text input clear button
+        b.go(f'/system/services#/?name=test')
```

Comment:
```
    not ok 2 /static-code/test-flake8
     # test/verify/check-system-services:1132:14: F541 f-string is missing placeholders
```

I need to force-push anyway as I restarted our CI bots and tests got stuck. I'll fix that while at it.